site/src/pages/CreateTokenPage/utils.ts

```diff
@@ -0,0 +1,73 @@
+import i18next from "i18next"
+
+export const NANO_HOUR = 3600000000000
```

Comment:
This is a bit of a pain - explanation [here](https://github.com/coder/coder/issues/6607).